web/src/app/pages/create-survey/data-collection/data-collection.module.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {NgModule} from '@angular/core';
+import {DataCollectionComponent} from './data-collection.component';
+import {MatButtonModule} from '@angular/material/button';
+import {ReactiveFormsModule} from '@angular/forms';
+import {MatFormFieldModule} from '@angular/material/form-field';
+import {MatRadioModule} from '@angular/material/radio';
+import {CommonModule} from '@angular/common';
+import {MatCardModule} from '@angular/material/card';
+
+@NgModule({
+  declarations: [DataCollectionComponent],
+  imports: [
+    CommonModule,
+    MatButtonModule,
+    MatCardModule,
+    MatFormFieldModule,
+    MatRadioModule,
+    ReactiveFormsModule,
+  ],
+  exports: [DataCollectionComponent],
+})
+export class DataCollectionModule {}
```

Comment:
Nit: Data collection in the context of Ground refers to the actual entry of field data by data collections. Can we rename this to something self-descriptive that doesn't collide with that name? 

In plain-English: "This component allows survey organizers to select who can specify data collection sites (i.e., locations of interest for which data is to be collected)." So maybe something like "LoiPermissionsComponent"?